eng/MSBuild/Packaging.props

```diff
@@ -4,7 +4,7 @@
     <SignArtifacts>$(SignAssembly)</SignArtifacts>
 
     <!-- Arcade SDK marks all but *.pkgproj as non-packable, we need to override. -->
-    <IsPackable>true</IsPackable>
+    <IsPackable Condition="'$(IsPackable)' == ''">true</IsPackable>
```

Comment:
I'm noticing in the msbuild.binlog that IsPackable is showing up as false for the transport.proj. I am unsure why, and would like to get some pointers. Not sure if this is the correct way to address it.

Reply:
I think you should undo this change. It's not necessary for your change.

Reply:
Yup, the change you are making is basically only respecting the original value in case it was already set, but if you are seeing false in the binlog, then that only means it is getting overridden later, so we should just keep this as it was.